cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/AddedOrRemovedBrokerParameters.java

```diff
@@ -38,6 +38,7 @@
   private boolean _dryRun;
   private boolean _throttleAddedOrRemovedBrokers;
   private boolean _skipHardGoalCheck;
+  private List<String> _replicaMovementStrategies;
```

Comment:
(this also applies to `DemoteBrokerParameters` and `RebalanceParameters`)
Can we store the `ReplicaMovementStrategy` here rather than `List<String>` -- i.e. looks like we always make the conversion to `ReplicaMovementStrategy` before we can use this parameter. Can we make this conversion in `ParameterUtils#getReplicaMovementStrategies` -- with a sanity check regarding whether the request is a `dryrun` or not? This also helps us centralize the conversion logic in one place.